### ISSUE TITLE
Docs page has broken links

### ISSUE THREAD

**Issue by georgewfraser** [fsharp/FSharp.Compiler.Service#828]
Many things on the right, for example:

http://fsharp.github.io/FSharp.Compiler.Service/reference/tokenizer.html
http://fsharp.github.io/FSharp.Compiler.Service/reference/devnotes.html

**Comment by dsyme** [comment]
Yep, though only if you are on a "reference" page.  All links from main pages such as http://fsharp.github.io/FSharp.Compiler.Service/ are ok
